week-3/Homework/mandatory/4-eligible-students.js

```diff
@@ -7,8 +7,12 @@
   - Returns an array containing onl the names of the who have attended AT LEAST 8 classes
 */
 
-function eligibleStudents() {
-
+function eligibleStudents(students) {
+  let studentsName = students.filter(([a, b]) => b >= 8);
+  let filterStudentsName = studentsName.map(x => x.slice(0, 1));
```

Comment:
Looking good! Since you're only getting 1 element from the array here, I think it would be simpler to just use array index syntax (`[...]`)?